src/test/regress/expected/local_dist_join_mixed.out

```diff
@@ -1612,3 +1615,11 @@ drop cascades to table unlogged_local
 drop cascades to materialized view mat_view
 drop cascades to view local_regular_view
 drop cascades to view dist_regular_view
+DEBUG:  drop cascades to view local_dist_join_mixed.dist_regular_view
```

Comment:
seems too verbose (and might be flaky) would be good to disable the logs at this point?